lesson1-the-bosun.md

```diff
@@ -46,35 +46,53 @@ Let's get started!
 ### Getting a Hosting Platform
 There are many options for hosting your Urbit, including doing so on your local MacOS computer, adding the Linux subsystem to your Windows box or even running your Urbit on a [Raspberry Pi](http://dasfeb.industries/). This guide will assume that you're choosing to run off of a Linux-based VPS hosted in the cloud, though it's very easy to translate these to other development platforms. There are several options that you can choose from for cloud-hosted Linux systems, but the most common are linked below.  Get one:
 
-**DigitalOcean**
-1. Create a [DigitalOcean Account and Add a Project and Droplet to your Account](https://www.digitalocean.com/docs/droplets/how-to/create/#:~:text=You%20can%20create%20one%20from,open%20the%20Droplet%20create%20page.)
+**Digital Ocean**
+1. Create a [Digital Ocean Account and Add a Project and Droplet to your Account](https://www.digitalocean.com/docs/droplets/how-to/create/#:~:text=You%20can%20create%20one%20from,open%20the%20Droplet%20create%20page.)
     * **NOTE:** You likely only need the the $10/mo (2GB RAM, 1 CPU, 50 GB SSD, 2 TB Transfer) and could even possibly get away with the $5/mo option, if you want to fiddle with the swap space.
 2. Generate [SSH Keys and Add Them to Your Droplet](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/)
 3. Connect to your [Droplet (potentially using Visual Studio Code - that's what I like!)](https://www.digitalocean.com/community/tutorials/how-to-use-visual-studio-code-for-remote-development-via-the-remote-ssh-plugin)
 
 **SSDNodes**
 1. Create a [SSDNodes Account and Create a Server](https://www.ssdnodes.com/)
     * **NOTE:** Again, you likely only need the minimal service they provide - choose as you see fit, though
-2. Generate [SSH Keys and Connect to Your SSDNode Securely](https://blog.ssdnodes.com/blog/connecting-vps-ssh-security/)
+2. Generate [SSH Keys and Connect to Your SSDNode Securely](https://blog.ssdnodes.com/blog/connecting-vps-ssh-security/**
 
 Once you have your hosted Linux solution in place and can log in as a non-root user to that system, proceed:
 
+*JAL(2021.3.2) I'd rather have this guide assume that you're running locally and
```

Comment:
Hi, just reading through this guide, and I agree. The easiest/cheapest version to get up and running seems to involve, for MacOS users, just doing this all locally.  Some instructions to that effect would be helpful for, I believe, a majority of readers. Could add this section in as a PR, if you'd like, as I'm planning to follow the tutorial locally on my mac.

Reply:
Hey - I didn't notice this earlier - I've converted the guide to working for local installs in any *nix system but did not specifically instruct on installation for Mac - instead I linked it out to the main urbit.org docs.